safeeyes/config/lang/pl.json

```diff
@@ -0,0 +1,51 @@
+{
+    "meta_info": {
+        "language_name": "Polski",
+        "language_name_en": "Polish"
+    },
+    "exercises": {
+        "short_break_close_eyes": "Mocno zaciśnij powieki",
+        "short_break_roll_eyes": "Ruszaj oczami z boku na bok",
+        "short_break_rotate_clockwise": "Wykonuj krążenia oczami zgodnie z ruchem wskazówek zegara",
+        "short_break_rotate_counter_clockwise": "Wykonuj krążenia oczami przeciwnie do ruchu wskazówek zegara",
+        "short_break_blink": "Pomrugaj oczami",
+        "short_break_focus_far_distance": "Skup wzrok na odległym punkcie",
+        "short_break_drink_water": "Napij się wody",
+        "long_break_walk": "Przejdź się chwilę",
+        "long_break_lean_back": "Oprzyj się wygodnie na krześle i zrelaksuj"
+    },
+    "messages": {
+        "ready_for_a_break": "Przygotuj się! Przerwa za {} sekund(y).",
+        "disabled_until_restart": "Wyłączony do ponownego uruchomienia",
+        "disabled_until_x": "Wyłączony do {}",
+        "next_break_at": "Następna przerwa o {}"
+    },
+    "ui_controls": {
+        "about": "O programie",
+        "audible_alert": "Powiadomienie dźwiękowe na koniec przerwy",
+        "cancel": "Anuluj",
+        "disable": "Zatrzymaj Safe Eyes",
+        "enable": "Uruchom Safe Eyes",
+        "enable_screen_lock": "Zablokuj ekran po każdej długiej przerwie",
+        "for_x_hour": "Na {} godzinę",
+        "for_x_hours": "Na {} godzin(y)",
+        "for_x_minutes": "Na {} minut(y)",
+        "idle_time": "Najkrótszy czas bez aktywności do wstrzymania odliczania (w minutach)",
+        "interval_between_two_breaks": "Czas między kolejnymi przerwami (w minutach)",
+        "language": "Język",
+        "long_break_duration": "Czas trwania długiej przerwy (w sekundach)",
+        "no_of_short_breaks_between_two_long_breaks": "Liczba krótkich przerw między długimi przerwami",
+        "postpone": "Odrocz",
+        "postpone_duration": "Odroczenie przerwy (w minutach)",
+        "quit": "Wyjdź",
+        "save": "Zapisz",
+        "settings": "Ustawienia",
+        "short_break_duration": "Czas trwania krótkiej przerwy (w sekundach)",
+        "skip": "Pomiń",
+        "strict_break": "Bezwzględna przerwa (ukryj przycisk pominięcia)",
```

Comment:
Google translates this as `Absolute pause (hide skip button)`. I feel it should be `Strict ...` instead of `Absolute`. English is not my mother tongue if both means same, please ignore this.